src/main/resources/views/CommercialVehicles/Defects.hbs

```diff
@@ -0,0 +1,39 @@
+{{~> views/CommercialVehicles/Defect
+    showDefectTitle=dangerous.defectsIsFirst
+    subTitle="Do not drive until repaired (dangerous defects)"
+    defects=dangerous.defects
+    defectsExists=dangerous.defectsExist
+    defectType="Dangerous"
+}}
+
+{{~> views/CommercialVehicles/Defect
+    showDefectTitle=major.defectsIsFirst
+    subTitle="Repair immediately (major defects)"
+    defects=major.defects
+    defectsExists=major.defectsExist
+    defectType="Major"
+}}
+
+{{~> views/CommercialVehicles/Defect
+    showDefectTitle=minor.defectsIsFirst
+    subTitle="Repair as soon as possible (minor defects)"
+    defects=minor.defects
+    defectsExists=minor.defectsExist
+    defectType="Minor"
+}}
+
+{{~> views/CommercialVehicles/Defect
+    showDefectTitle=advisory.defectsIsFirst
+    subTitle="Monitor and repair if necessary (advisory defects)"
```

Comment:
I'm doing a comparison with the PDF on the ticket, and it looks like this heading should be 

`Monitor and repair if necessary (advisories)` rather than `(advisory defects)`